nixos/server/default.nix

```
@@ -111,10 +111,56 @@
     '';
   };
 
-  # use TCP BBR has significantly increased throughput and reduced latency for connections
+  # https://www.kernel.org/doc/html/latest/networking/ip-sysctl.html
```

@randomizedcoder The optimization you applied, maybe we should agree first what usecase we are optimizing for. Could you maybe describe what type of server/client setup you have based your considerations on? I would like to add this as a comment for future readers.

Something like that (feel free to change it what you want to)
```suggestion
   # These settings optimize network configuration for servers with X-Y Gpbs and serving clients with Xms latency
  # https://www.kernel.org/doc/html/latest/networking/ip-sysctl.html
```